www/front_src/src/Resources/Graph/Performance/GraphActions.tsx

```diff
@@ -64,7 +69,7 @@ const GraphActions = ({
   const [exporting, setExporting] = useState<boolean>(false);
   const { format } = useLocaleDateTimeFormat();
   const navigate = useNavigate();
-
+  const isResourceAD = equals(resourceType, ResourceType.anomalydetection);
```

Comment:
```suggestion
  const isResourceAnomalyDetection = equals(resourceType, ResourceType.anomalydetection);
```

Avoid using shortcuts in naming